integration/presto/pom.xml

```diff
@@ -31,7 +31,7 @@
   <packaging>presto-plugin</packaging>
 
   <properties>
-    <presto.version>0.186</presto.version>
+    <presto.version>0.187</presto.version>
```

Comment:
two question :1. as per mailing list discussed, the version of presto be suggested with 0.186, here changes to 0.187 , i think we should have a stable and fixed version number,it might not be good if carbon1.3 optionally upgrade version number for presto integration module. 2. if users still use the old presto version, whether it works , or not ?

Reply:
Yes I think we need to specify 0.187 version upgrade in another mailing list, with 0.187 in carbon it works with 0.186 but not with 0.166 as the interfaces for Connector is changed in 0.186

Reply:
for 1.3.0, we have to finalize the presto version.